backend/danswer/db/tag.py

```diff
@@ -113,6 +114,8 @@ def get_tags_by_value_prefix_for_source_types(
     sources: list[DocumentSource] | None,
     limit: int | None,
     db_session: Session,
+    # if set, both tag_key_prefix and tag_value_prefix must be a match
```

Comment:
Does the name of this function make sense since it's checking key or value or both?